R/as_gt.R

```diff
@@ -78,61 +78,21 @@ as_gt <- function(x, ...) {
 #' ) %>%
 #'   summary() %>%
 #'   as_gt()
-as_gt.fixed_design <- function(x, title = NULL, footnote = NULL, ...) {
-  method <- fd_method(x)
+as_gt.fixed_design_summary <- function(x, title = NULL, footnote = NULL, ...) {
+  if (is.null(title)) title <- attr(x, "title", exact = TRUE)
```

Comment:
I suggest that we provide a helper function like this in utils.R: https://github.com/yihui/knitr/blob/35308a212e15649b8d191d652714abd9108f7c87/R/utils.R#L51 The only thing I can't tolerate about R is the default partial matching.

Reply:
Good idea. Done in https://github.com/Merck/gsDesign2/pull/497/commits/1964661bf7b8c9a9d021d0383f559840db60e25a